test/libsolidity/lsp/references/import_aliases.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
In all these tests, can you also add instances of referenced names within strings and comments (including the SPDX comment)?